src/utils/tests/expression.js

```diff
@@ -0,0 +1,67 @@
+import fromJS from "../fromJS";
+
+jest.mock("../scopes");
+jest.mock("../parser");
```

Comment:
from my tests it needs to be here so that it's properly mocked when expression.js got imported.

I'm not sure how this interacts with other files. Maybe Jest is smart and run all files separately and everything will be fine.